src/stylesheets/screens/question.scss

```diff
@@ -0,0 +1,69 @@
+.questions {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+    height: rem(60px);
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-right {
+    order: 2;
+    margin-left: auto;
+  }
+
+  &__container {
+    @include details-container();
+    position: relative;
+
+    flex: 1 1 auto;
+    height: calc(100vh - 135px);
+  }
+
+  &__details {
+    margin: auto 0;
+  }
+
+  &__number {
+    color: map-get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'sm');
+    line-height: rem(20px);
+
+    opacity: 0.5;
+  }
+
+  &__title {
+    margin: 0;
+    padding-top: rem(10px);
+
+    color: map-get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'xxxl');
+    line-height: rem(41px);
+  }
+
+  &__footer {
+    display: flex;
+    flex: 0 1 auto;
+    width: 100%;
+    height: rem(75px);
+    padding: rem(0 16px 0);
```

Comment:
@rafayet-monon What if we separate this?
```css
padding-right: rem(16px);
padding-left: rem(16px);
```

Reply:
fixed in [2387eb2](https://github.com/rafayet-monon/surveyor/pull/19/commits/2387eb2ef2dc3009ecac3a222d219698f2ed0864)